.github/workflows/release.yml

```diff
@@ -88,20 +82,20 @@ jobs:
           echo '#!/bin/sh
           ci/bin/unlock-agent.sh
           mkdir -p /go/src
-          go mod tidy
-          go mod vendor
+          GO111MODULE=on go mod tidy
+          GO111MODULE=on go mod vendor
```

Comment:
This seems weird. L98 below sets it to off, which should be the correct value.

Reply:
We need this, because this is run inside the container which runs with `GO111MODULE=off` as set in L98.
If it isn't set to to `on` for  these two statements, the vendored dependencies are not downloaded(because it's running it with GO111MODULE set to off)

Reply:
Please write up this logic in https://tyktech.atlassian.net/wiki/spaces/EN/pages/449708061/Release+Engineering#Gateway-specific-workflows

Reply:
Or in a template comment.